aea/connections/stub/connection.py

```diff
@@ -143,13 +143,11 @@ def read_envelopes(self) -> None:
         """Receive new envelopes, if any."""
         line = self.input_file.readline()
         logger.debug("read line: {!r}".format(line))
+        lines = b""
```

Comment:
@marcofavorito I had to change this implementation. It turns out the old one was problematic with protobuf as protobuf encoded messages could contain a newline character causing this logic not to work. Now we are constrained to single envelopes, but I think this is ok as the connection will pick them off the file very quickly